packages/picasso.js/src/core/chart-components/text/text.js

```diff
@@ -47,6 +47,9 @@ function generateTitle({ title, definitionSettings, dock, rect, measureText, sty
     dy: 0,
     anchor: getTextAnchor(dock, definitionSettings.anchor),
     baseline: 'alphabetical',
+    stroke: style.stroke || 'transparent',
+    strokeWidth: style.strokeWidth || 0,
+    fontWeight: style.fontWeight || 'normal',
```

Comment:
could this not be only
```suggestion
    stroke: 'transparent',
    strokeWidth: 0,
    fontWeight: 'normal',
```


the values from style should be handled by
```
extend(struct, style.text);
```

Reply:
Agreed !